src/ducks/transactions/Transactions.jsx

```diff
@@ -129,35 +126,60 @@ const TableTrNoDesktop = translate()(
                   getCategoryId(transaction)
                 )}`
               )}
+              onClick={() => selectTransaction(transaction)}
             >
               <CategoryIcon
                 category={getParentCategory(getCategoryId(transaction))}
               />
             </Img>
             <Bd className="u-mr-half u-ellipsis">
-              <ListItemText
-                primaryText={getLabel(transaction)}
-                secondaryText={
-                  !filteringOnAccount && getAccountLabel(transaction.account)
-                }
-              />
+              <div onClick={() => selectTransaction(transaction)}>
+                <ListItemText
```

Comment:
ListItemText does not provide onClick ?

Reply:
if it does not can you open an issue ?

Reply:
The best would be that `<Bd />` propagates all its props to its root element. I did a PR on cozy-ui for that : https://github.com/cozy/cozy-ui/pull/441

Reply:
Actually ListItemText doesn't spread its props either

Reply:
Now it does :)